recipes/newmat/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,11 @@
+#include "newmatap.h"
+
+#ifdef use_namespace
+using namespace RBD_LIBRARIES;
+#endif
+
```

Comment:
```suggestion
```
The source in test package should be configured or from the package or by the own test package. In this case, the definition `use_namespace` is not configurable and has no effect. Please, remove it.